spec/system/dashboard/callouts_spec.rb

```diff
@@ -17,6 +17,8 @@
       )
     end
 
+    expect(page).to have_css("i.fa-key")
```

Comment:
This really should be in here https://github.com/somleng/somleng-scfm/blob/develop/spec/system/dashboard/dashboard_spec.rb